### PR TITLE
Remove chart dependencies

### PR DESCRIPTION
**Pull request by maruiz93**
Install `knative` and `cert-manager` operators as a previous step with `kubectl`.

Add a known issue for handling the race conditions in the docs.

### CODE REVIEW

**Comment by maruiz93** [comment]
I've tested the whole installation process with this script:

```bash
#!/bin/sh

CLUSTER_NAME=kind-cluster
export KO_DOCKER_REPO=kind.local
export KIND_CLUSTER_NAME=${CLUSTER_NAME} # Defaults to "kind"

kind delete cluster -n ${CLUSTER_NAME}
kind create cluster -n ${CLUSTER_NAME}

cmd/operator/generate.sh

export KNATIVE_EVENTING_VERSION=v1.14.3
export CERT_MANAGER_VERSION=v1.9.1

kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-crds.yaml
kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml

kubectl rollout -n cert-manager status deployment cert-manager-webhook

helm install kubearchive charts/kubearchive --namespace kubearchive --create-namespace \
    --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api) \
    --set-string sink.image=$(ko build github.com/kubearchive/kubearchive/cmd/sink) \
    --set-string operator.image=$(ko build github.com/kubearchive/kubearchive/cmd/operator)

```

**Review by rh-hemartin — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -127,9 +127,14 @@ go test -v ./... -tags=integration
...
-1. Build the dependencies of the chart
+1. Install knative-events and cert-manager operators
    ```bash
-   helm dependency build charts/kubearchive
+   export CERT_MANAGER_VERSION=v1.9.1
+   export KNATIVE_EVENTING_VERSION=v1.14.3
+  
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml 
+   kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-crds.yaml
+   kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
```
Comment (rh-hemartin) [inline DEVELOPMENT.md:137]: Why is this inside the "remote IDE debugging" section? This should be after the cluster creation.

Comment (maruiz93) [reply DEVELOPMENT.md]: yes you are right, should be fixed now

**Review by ggallen — COMMENTED** [review]
I propose we go back to my original solution in https://github.com/kubearchive/kubearchive/pull/95.  It does all of this cleanly without the need for extra steps for the user.

**Review by rh-hemartin — APPROVED** [review]

**Comment by maruiz93** [comment]
After a discussion with the team, the pre-hook solution with the installation of the `knative` and `cert-manager` operators as crds of the kubearchive is preferred. 

Closing this PR in favor of https://github.com/kubearchive/kubearchive/pull/95

**Pull request closed without merging**